### PR TITLE
New package: sumo-0.32.0

### PR DESCRIPTION
**Pull request by warrior-graph**
http://sumo.sourceforge.net/

### CODE REVIEW

File: srcpkgs/sumo/template [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+build_style=gnu-configure
+hostmakedepends="pkg-config libgdal-tools"
+makedepends="fox-devel libgdal-devel libxerces-c-devel proj-devel"
+short_desc="SUMO (Simulation of Urban MObility) and included applications"
+maintainer="Warrior Graph <marques@larces.uece.br>"
+license="EPL-2.0"
+homepage="http://sumo.sourceforge.net/"
+distfiles="${SOURCEFORGE_SITE}/${pkgname}/${pkgname}-src-0.32.0.tar.gz"
+checksum=00753ca57a9911f0c99202505a6b05b1777168134842d7924fd827766642608a
+
```
Comment (maxice8) [inline srcpkgs/sumo/template]: unnecessary extra blank lines

Comment (warrior-graph) [reply srcpkgs/sumo/template]: I'm a little bit confused about the difference between **hostmakedepends** and **makedepends**. I hope it's right.

**Comment by maxice8** [comment]
everything in one commit

**Comment by warrior-graph** [comment]
It's my first time making a package. Can you explain this errors that I got on Travis CI? Thank you

**Comment by warrior-graph** [comment]
I was using gcc 7.x and I figure out that it doesn't compile with gcc 8.2.0 because of:

/usr/include/c++/8.2/bits/stl_tree.h:452:21: error: static assertion failed: comparison object must be invocable with two arguments of key type
       static_assert(__is_invocable<_Compare&, const _Key&, const _Key&>{},

**Comment by Johnnynator** [comment]
@warrior-graph Searching for gcc 8 in the sumo repo brings up this patch https://github.com/eclipse/sumo/commit/00b3de97f6ea8ae47681d3549fe877d9416d7a56 which should fix this issue.

**Comment by Johnnynator** [comment]
I quickly backported the patch https://gist.github.com/Johnnynator/e2597360067e1f70551a01737d075c6a .
I have done no runtime tests so far and it is still compiling locally.

**Comment by warrior-graph** [comment]
Thank you. I applied the patch and everything works fine. I will open a new pull request.

**Comment by maxice8** [comment]
please do not open a new pull request just force-push the changes

**Pull request closed without merging**